config/application.rb

```diff
@@ -62,6 +62,7 @@ class Application < Rails::Application
 
     # Emails
     config.email_test_address = ENV["EMAIL_TEST_ADDRESS"]
+    config.second_renewal_email_reminder_days = ENV["SECOND_RENEWAL_EMAIL_BEFORE_DAYS"]
```

Comment:
We have a default value set for the first email reminder and the grace window. I think we also need a default value set here of `14`.

```suggestion
    config.second_renewal_email_reminder_days = ENV["SECOND_RENEWAL_EMAIL_BEFORE_DAYS"] || 14
```

When we release we will be overriding this to `7`, but we intend to return to the default of 14 days once the registration peak is at an end.

Reply:
I don't see any default for the first email reminder and grace window 🤔 will update this anyway

Reply:
Oh. I rebased XD